custom_components/localtuya/pytuya/__init__.py

```diff
@@ -301,40 +343,20 @@ def generate_payload(self, command, data=None):
             json_data["dps"] = data
         if command_hb == "0d":
```

Comment:
Now this should be
`if command_hb == 0x0d:`
It took a lot to figure it out!!

Reply:
Yeah, moved from strings to integer instead where possible.